js/Slider.js

```diff
@@ -249,22 +249,16 @@ const Slider = (
       ref={forwardedRef}
       style={style}
       onChange={onChangeEvent}
-      onSlidingStart={onSlidingStartEvent}
-      onSlidingComplete={onSlidingCompleteEvent}
-      onValueChange={onValueChangeEvent}
+      onRNCSliderSlidingStart={onSlidingStartEvent}
+      onRNCSliderSlidingComplete={onSlidingCompleteEvent}
+      onRNCSliderValueChange={onValueChangeEvent}
```

Comment:
can't we avoid changing public API?

Reply:
I haven't changed a public API, we still expose `onSlidingStartEvent `,  `onSlidingCompleteEvent` and `onValueChangeEvent` and use this class to change props names under the hood. W/o this change the app throws an error (built by xcode)

Reply:
Oh, that's great!